src/main/java/duke/main/Duke.java

```diff
@@ -0,0 +1,114 @@
+package duke.main;
+
+
+import java.util.ArrayList;
+import java.io.IOException;
+import java.util.Scanner;
+
+import duke.command.*;
+import duke.command.actionCommands.DeleteTask;
+import duke.command.actionCommands.ListTasks;
+import duke.command.actionCommands.MarkTask;
+import duke.command.actionCommands.UnmarkTask;
+import duke.command.taskCommands.DeadlineTask;
+import duke.command.taskCommands.EventTask;
+import duke.command.taskCommands.TodoTask;
+import duke.exception.InvalidTaskException;
+import duke.tasks.*;
+
+import static duke.fileOperation.FileOperation.*;
+import static java.lang.System.exit;
+
+public class Duke {
+    static final int limitTask = 100;
+    //    static Task[] tasksList = new Task[limitTask];
```

Comment:
Consider removing this line of commented code if you no longer plan to use/test it out.